Makefile

```diff
@@ -237,6 +237,13 @@ build_tools/closure.jar: build_tools/.closure_compiler_version
 	wget -P build_tools https://github.com/mykmelez/closure-compiler/releases/download/$(CLOSURE_COMPILER_VERSION)/closure.jar
 	touch build_tools/closure.jar
 
+build_tools/emsdk-portable.tar.gz:
+	wget -P build_tools https://s3.amazonaws.com/mozilla-games/emscripten/releases/emsdk-portable.tar.gz
```

Comment:
> We should probably host this ourselves, so we can control the version.

The Portable SDK supports installing older versions of the SDK and toolchains, so we can control their version (although there may be something in the Portable SDK that could still break in an update).